Clinical-Genomics/cg
# Fix auto upload

This PR fixes the missing cases in cli command cg analysis auto.

How to test:
0. install branch fix-auto-upload on stage on rasta
1. run `sh ~/servers/resources/clinical-db.scilifelab.se/dbcopy-prod-to-stage.sh cg` on clinical-db
1. `us`
1. run `cg status analysis` / `cg analysis auto`

Expected outcome:
All the analyses ready for MIP analysis are listed/started
